app/_locales/th/messages.json

```diff
@@ -136,32 +136,57 @@
     "message": "เสร็จสิ้น"
   },
   "downloadSecretBackup": {
-    "message": "ดาวน์โหลด Phrase แบ็คอัพลับนี้และเก็บรักษาไว้ให้ปลอดภัยในฮาร์ดดิสก์หรืออุปกรณ์เก็บข้อมูลภายนอกที่มีการเข้ารหัส"
+    "message": "ดาวน์โหลด Secret Recovery Phrase และเก็บรักษาไว้ให้ปลอดภัยในฮาร์ดดิสก์หรืออุปกรณ์เก็บข้อมูลภายนอกที่มีการเข้ารหัส"
   },
   "downloadStateLogs": {
-    "message": "ดาวน์โหลดล็อกสถานะ"
+    "message": "ดาวน์โหลด logs สถานะ"
   },
   "edit": {
     "message": "แก้ไข"
   },
   "editContact": {
-    "message": "แก้ไขผู้ติดต่อ"
+    "message": "แก้ไขรายชื่อผู้ติดต่อ"
   },
   "endOfFlowMessage1": {
-    "message": "คุณผ่านการทดสอบแล้ว กรุณารับผิดชอบในการรักษา Seed Phrase ให้ปลอดภัย"
+    "message": "คุณผ่านการทดสอบแล้ว โปรดเก็บรักษา Secret Recovery Phrase ของคุณให้ปลอดภัย ถือเป็นความรับผิดชอบของคุณ!"
+  },
+  "endOfFlowMessage2": {
+    "message": "เคล็ดลับในการจัดเก็บอย่างปลอดภัย"
+  },
+  "endOfFlowMessage3": {
+    "message": "เก็บสำรองไว้ในที่ปลอดภัยหลาย ๆ ที่"
+  },
+  "endOfFlowMessage4": {
+    "message": "อย่าให้คนอื่นรู้ Secret Recovery Phrase ของคุณ"
   },
   "endOfFlowMessage5": {
-    "message": "โปรดระวังฟิชชิ่ง MetaMask จะไม่ขอ Seed Phrase ของคุณโดยไม่มีการแจ้งล่วงหน้าเลย"
+    "message": "โปรดระวังฟิชชิ่ง! โดยปกติแล้ว MetaMask จะไม่ขอ Secret Recovery Phrase ของคุณ"
+  },
+  "endOfFlowMessage6": {
+    "message": "หากคุณต้องการตรวจสอบ Secret Recovery Phrase อีกครั้ง คุณสามารถตรวจสอบได้ที่เมนู การตั้งค่า -> ความปลอดภัย"
+  },
+  "endOfFlowMessage7": {
+    "message": "หากคุณมีคข้อสงสัยหรือเห็นสิ่งผิดปกติโปรดติดต่อฝ่ายสนับสนุนของเรา $1",
+    "description": "$1 is a clickable link with text defined by the 'here' key. The link will open to a form where users can file support tickets."
+  },
+  "endOfFlowMessage8": {
+    "message": "MetaMask ไม่สามารถกู้คืน Secret Recovery Phrase ให้คุณได้"
   },
   "endOfFlowMessage9": {
     "message": "เรียนรู้เพิ่มเติม"
   },
+  "endOfFlowMessage10": {
+    "message": "เสร็จหมดแล้ว"
+  },
```

Comment:
@EKAPHONG lint is failing because this message should be after endOfFlowMessage1 ... if you run `yarn lint:fix` it should resolve it for you.

Reply:
Oh! thank you so much I fixed it in d3fa15d095ff20044ff900cd9dc68c58d601e5e1